packages/beacon-node/test/sim/mergemock.test.ts

```diff
@@ -183,7 +202,22 @@ describe("executionEngine / ExecutionEngineHttp", function () {
       await Promise.all(validators.map((v) => v.close()));
     });
 
+    let engineBlocks = 0;
+    let builderBlocks = 0;
     await new Promise<void>((resolve, _reject) => {
+      bn.chain.emitter.on(routes.events.EventType.block, async (blockData) => {
+        const {data: fullOrBlindedBlock} = (await bn.api.beacon.getBlockV2(blockData.block).catch((_e) => null)) ?? {};
```

Comment:
If there's a block event there must be a block available in the API right? This catch all to null seems dangerous

Reply:
it is to just update the stats, but nevertheless removed the catch :+1: